src/main/java/org/folio/reshare/index/storage/Storage.java

```diff
@@ -24,8 +24,15 @@
 public class Storage {
   private static final Logger log = LogManager.getLogger(Storage.class);
 
+  private static final String IDENTIFIER_TYPE_ID_ISBN = "8261054f-be78-422d-bd51-4ed9f33c3422";
```

Comment:
These constants should be factored out to individual match key method implementation classes.